src/CosmosDB/CosmosDB/CosmosDBAccount/GetAzCosmosDBAccountKey.cs

```diff
@@ -69,7 +72,7 @@ public override void ExecuteCmdlet()
             if (Type.Equals("ConnectionStrings", StringComparison.OrdinalIgnoreCase))
             {
                 DatabaseAccountListConnectionStringsResult response = CosmosDBManagementClient.DatabaseAccounts.ListConnectionStringsWithHttpMessagesAsync(ResourceGroupName, Name).GetAwaiter().GetResult().Body;
-                WriteObject(new PSDatabaseAccountListKeys(response).Keys);
+                WriteObject(new PSDatabaseAccountListKeys(response).ConnectionStrings);
```

Comment:
This is a breaking change as the types are different. We only allow it to be released in breaking change release sprint. May I know if you have any reason to release the breaking change? The [GenericBreakingChange] attribute is only used as a declaration.

Reply:
@carjackson-msft The next breaking change release is around Ignite 2023. The exact date is not determined yet. You can refer to our milestone https://github.com/Azure/azure-powershell/milestones and we will keep it up to date.

Reply:
@carjackson-msft Sorry I cannot approve the PR and merge the code. Can you please temporarily close this PR or make it as a draft for now? During the breaking change release, you can re-open to have it reviewed and merged.